### PR TITLE
bug fix: User details updation in usermanagement tab

### PR DESCRIPTION
**Pull request by SarthakJain26**
Signed-off-by: Sarthak Jain <sarthak.jain@mayadata.io>

## Proposed changes
- Updated User Info in the User management tab.
- Fixed search bar issue
- Changed 'name' heading to 'status' in User management table

Summarize your changes here to communicate with the maintainers and make sure to put the link of that issue

## Types of changes
- Bug fixes

What types of changes does your code introduce to Litmus? Put an `x` in the boxes that apply
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation Update (if none of the other choices applies)

## Checklist

Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.
- [x] I have read the [CONTRIBUTING](https://github.com/litmuschaos/litmus/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the commit for DCO to be passed.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have added necessary documentation (if appropriate)

## Dependency
- Please add the links to the dependent PR need to be merged before this (if any).

### CODE REVIEW

**Review by arkajyotiMukherjee — APPROVED** [review]
LGTM

**Review by Saranya-jena — APPROVED** [review]
LGTM

File: litmus-portal/frontend/src/views/Settings/AccountsTab/PersonalDetails/index.tsx [hunk 1]
```diff
@@ -130,7 +150,6 @@ const PersonalDetails: React.FC = () => {
             {error.length ? (
               <div className={classes.errDiv}>
                 {/* <img src="./icons/checkmark.svg" alt="checkmark" /> */}
-
                 <div className={classes.textError}>
                   <Typography className={classes.typo} align="center">
                     <strong> Error </strong> while updating details.
```
Comment (oumkale) [inline litmus-portal/frontend/src/views/Settings/AccountsTab/PersonalDetails/index.tsx:155]: Can you keep this in translation

File: litmus-portal/frontend/src/views/Settings/UserManagementTab/UserManagement/index.tsx [hunk 2]
```diff
@@ -232,7 +235,7 @@ const UserManagement: React.FC = () => {
                       <TableHead>
                         <TableRow className={classes.TR}>
                           <StyledTableCell className={classes.styledTC}>
-                            Name
+                            Status
                           </StyledTableCell>
                           <StyledTableCell>Username</StyledTableCell>
```
Comment (oumkale) [inline litmus-portal/frontend/src/views/Settings/UserManagementTab/UserManagement/index.tsx:240]: same

**Comment by JitendraDhakar** [comment]
A good

**Review by Saranya-jena — APPROVED** [review]
LGTM

**Review by amityt — APPROVED** [review]

**Pull request merged**